pkg/portal/prometheus/prometheus.go

```diff
@@ -48,10 +48,14 @@ func New(baseLog *logrus.Entry,
 	}
 
 	aadAuthenticatedRouter.NewRoute().Path("/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/microsoft.redhatopenshift/openshiftclusters/{resourceName}/prometheus").HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		r.URL.Path += "/"
+		r.URL.Path += "/graph"
```

Comment:
This is changing all Prometheus routes to `/graph`, but it has to be based on the cluster version. Only 4.11 requires the appended `/graph`, so all other clusters in the fleet won't work with this blanket change.